pyproject.toml

```diff
@@ -73,6 +73,11 @@ aggressive = 3
 [tool.ruff]
 line-length = 99
 target-version = "py38"
+exclude = ["docs"]
```

Comment:
We should still exclude everything that's excluded by default.

Reply:
From the Ruff docs: "Note that you'll typically want to use [extend-exclude](https://docs.astral.sh/ruff/settings/#extend-exclude) to modify the excluded paths." https://docs.astral.sh/ruff/settings/#extend-exclude